2nd-gen/delete-unused-accounts-cron/functions/index.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+// [START all]
+// [START import]
+// The Cloud Functions for Firebase SDK to create v2 Cloud Functions and setup triggers.
+const { onSchedule } = require('firebase-functions/v2/scheduler');
+const { logger } = require('firebase-functions');
+
+// The Firebase Admin SDK to delete inactive users.
+const admin = require('firebase-admin');
+admin.initializeApp();
+
+// The es6-promise-pool to limit the concurrency of Promise's.
```

Comment:
You don't need the apostrophe -- nor the capital "P" I think, since this isn't a literal here.

Reply:
thanks, fixed